src/In.ProjectEKA.HipService/Consent/ConsentController.cs

```diff
@@ -22,18 +22,28 @@ public ConsentController(IConsentRepository consentRepository)
             [FromHeader(Name = "X-ConsentManagerID")] string consentManagerId,
             [FromBody] ConsentArtefactRequest consentArtefactRequest)
         {
-            var consent = new Consent(consentArtefactRequest.ConsentDetail.ConsentId,
+            var consent = new Consent(null,
                 consentArtefactRequest.ConsentDetail,
                 consentArtefactRequest.Signature,
                 consentArtefactRequest.Status,
                 consentManagerId);
-            if (consentArtefactRequest.Status == ConsentStatus.GRANTED)
+            ;
+            switch (@consentArtefactRequest.Status)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'default' clause to this 'switch' statement.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5392701)